Mathlib/Data/Fin/Tuple/Finset.lean

```diff
@@ -57,33 +57,48 @@ namespace Finset
 variable {n : ℕ} {α : Fin (n + 1) → Type*} {p : Fin (n + 1)} (S : ∀ i, Finset (α i))
 
 lemma map_consEquiv_filter_piFinset (P : (∀ i, α (succ i)) → Prop) [DecidablePred P] :
-    ((piFinset S).filter fun r ↦ P <| tail r).map (consEquiv α).symm.toEmbedding =
-      S 0 ×ˢ (piFinset fun x ↦ S <| succ x).filter P := by
+    {r ∈ piFinset S | P (tail r)}.map (consEquiv α).symm.toEmbedding =
```

Comment:
Is this an equality of `finset` (as in the original version) — I mean, is there a new notation for `filter` — or is this an equality of sets ?

Reply:
Yes, this is a new notation for `filter`. See #11532